src/main/java/com/stripe/model/checkout/Session.java

```diff
@@ -489,6 +489,12 @@ public LineItemCollection listLineItems(SessionListLineItemsParams params, Reque
     @SerializedName("acss_debit")
     AcssDebit acssDebit;
 
+    @SerializedName("boleto")
+    PaymentIntent.PaymentMethodOptions.Boleto boleto;
+
+    @SerializedName("oxxo")
+    PaymentIntent.PaymentMethodOptions.Oxxo oxxo;
+
```

Comment:
Those two are wrong. This should not be shared with the `PaymentIntent` class' options. It should be a separate resource internally in pay-server so that when they diverge over time we don't cause breaking changes. This will require changes internally first.

Reply:
Great catch. Thanks Remi!